docs/concepts/trading-framework/market-lifecycle.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_position: 7
+title: Market lifecycle
+hide_title: false
+---
+The market lifecycle begins when a proposal for a new market is accepted, and is driven by the market's state. The table shows a high-level view of each part of a market's lifecycle, including when that stage is over and a new one begins. Below, learn more about each lifecycle stage.
+
+| Status              | Accepting LP[1]  | Trading Mode        | Condition for entry                                                           | Condition for exit                   
+| ------------------ | -------------- | ------------------- | --------------------------------------------------------------- | -----------------------------------------         
+| Proposed           |   Yes          | No trading          | Governance proposal valid and accepted                                       | Governance proposal voting period ends
+| Rejected           |   No           | No trading          | Outcome of governance votes is to reject the market             | N/A                                                    
+| Pending            |   Yes          | Opening auction     | Governance vote passes                                     | Governance vote (to close) OR enactment date reached
+| Cancelled           |  No           | No trading          | Market triggers cancellation condition or governance votes to close before market becomes Active              | N/A                                                    
```

Comment:
This is mentioned in specs but does not, in fact, exist. There is a workaround for "cancelling" markets by editing oracle and submitting - so not sure it's coming any time soon.

Reply:
Ugh - thought I got rid of that, thank you